submissions/emmanuelEsuong/css/styles-mediaQuery.css

```diff
@@ -0,0 +1,264 @@
+/* media-query (tablet)*/
+@media screen and (min-width: 765px) and (max-width: 900px) {
+        /* signUp page */
+    /* formfield */
+    #s-form {
+        border-radius: 25px;
+        border: solid 1px;
+        padding: 35px 20px 20px 20px;
+        border: 2px 2px 2px 2px
+    }
+    /* signupform */
+    #form {
+        overflow: hidden;
+        margin-top: 10px;
+        float: right;
+        margin-right: 5.5cm;
+    }
+    #form input {
+        width: 550px;
+        padding: 10px;
+        border-radius: 7px;
+        border: solid 1px;
+    }
+    #table, #form {
+        display: inline-block;
+    }
+    #form #terms {
+        width: 50px;
+        float: left;
+        margin-left: 2cm;
+    }
+    #termsLabel {
+        color: rgb(2, 209, 2);
+        font-weight: bold;
+    }
+    button {
+        border-radius: 3px;
+        border: 0px;
+        margin-left: 220px;
+        padding: 7px;
+        padding-left: 20px;
+        padding-right: 20px;
+        background-color: rgb(2, 209, 2);
+        font-weight: bold;
+        color: white;
+    }  
+    ul {
+        list-style-type: none;
+        padding: 0px;
+        overflow: hidden;
+    }
+    /* registration info */
+    #regis {
+        font-family: Arial, Helvetica, sans-serif;
+        font-size: 40px;
+        font-weight: bold;
+        color: rgb(2, 209, 2);
+    }
+    #rg, #rgs, #gt {
```

Comment:
The naming here is not explanatory, kindly update

Reply:
Updated.